src/main/java/com/google/devtools/build/lib/remote/options/RemoteOptions.java

```diff
@@ -309,13 +318,25 @@ public RemoteOutputsStrategyConverter() {
       defaultValue = "",
       documentationCategory = OptionDocumentationCategory.REMOTE,
       effectTags = {OptionEffectTag.UNKNOWN},
+      deprecationWarning = "--remote_default_platform_properties has been deprecated in favor of --remote_default_exec_properties.",
       help =
           "Set the default platform properties to be set for the remote execution API, "
               + "if the execution platform does not already set remote_execution_properties. "
               + "This value will also be used if the host platform is selected as the execution "
               + "platform for remote execution.")
   public String remoteDefaultPlatformProperties;
 
+  @Option(
+      name = "remote_default_exec_properties",
```

Comment:
Do we really need "remote_default" prefix for this flag? wouldn't it be simpler just `exec_properties`?

Reply:
I think just exec_properties is misleading. It's important to state that this flag is a default value that's overriden by platforms. Theoretically we could drop the "remote_" prefix, but then again we have it for all the other remote options as well and in case exec_properties would be introduced for a different execution strategy some time in the future it would for sure require a different default_exec_properties flag given that the values are really specific to the execution environment.

So I think we should keep the naming, well aware that the flag name is a bit long.

Reply:
Alright, fair enough.
The only thing - I don't think it's clear from the current description that 
> this flag is a default value that's overriden by platforms

Could you then just adjust the description?

Reply:
> Set the default exec properties to be used as the remote execution platform if an execution platform does not already set exec_properties.

How would you like me to rephrase it?

Reply:
> Set the default exec properties to be used as the remote execution platform if an execution platform does not already set exec_properties.

Ah, sorry, good enough, missed this :confused: